framework/src/media/audio/audio_manager.c

```diff
@@ -217,54 +217,64 @@ audio_manager_result_t find_audio_card(audio_card_type_t card_type)
 	return ret;
 }
 
+static audio_manager_result_t find_audio_card_if_empty(audio_card_type_t card_type)
```

Comment:
It looks that considering INPUT card_type is needed.

Reply:
Fixed